notebooks/deployment-manager/create_notebook.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2019 Google Inc. All rights reserved.
+# Copyright 2019 Google Inc. All Rights Reserved
```

Comment:
Now it's just a game.

Reply:
Hehehe, yeah, this is a never ending challenge.   :)
I think it is really done this time.